definitions/tools/pvacseq.cwl

```diff
@@ -15,14 +15,14 @@ arguments: [
     { valueFrom: " && ", shellQuote: false },
     "/usr/local/bin/pvacseq", "run",
     "--iedb-install-directory", "/opt/iedb",
-    "--blastp-path", "/opt/ncbi-blast-2.12.0+/bin/blastp",
+    "--peptide-fasta", "/opt/reference_fasta/Homo_sapiens.GRCh38.pep.all.fa.gz",
```

Comment:
Would it make sense for this to be an input the user could specify but with this path as the default?